core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -804,12 +817,18 @@ private[spark] class AppStatusListener(
 
   override def onStageExecutorMetrics(executorMetrics: SparkListenerStageExecutorMetrics): Unit = {
     val now = System.nanoTime()
-
+    Option(liveStages.get((executorMetrics.stageId, executorMetrics.stageAttemptId)))
+      .foreach { stage =>
+      stage.peakExecutorMetrics.compareAndUpdatePeakValues(executorMetrics.executorMetrics)
```

Comment:
nit: Is it possible to change the name of the parameter or the field member? executorMetrics.executorMetrics feels kind of weird

Reply:
How about "metrics", or "stageExecutorMetrics" for the parameter name?

Reply:
yeah, sounds good to me.